docs/topics/conditional-view-processing.txt

```diff
@@ -65,6 +65,8 @@ order, as the view function they are helping to wrap. The function passed
 last time the resource was modified, or ``None`` if the resource doesn't
 exist. The function passed to the ``etag`` decorator should return a string
 representing the `ETag`_ for the resource, or ``None`` if it doesn't exist.
+Additionally adds the ``ETag`` and ``Last-Modified`` headers to the response if
+they exist, are not already set by the view and the request method is safe.
```

Comment:
Documentation should use complete sentences. Maybe: "The decorator also adds the the ``ETag`` and ``Last-Modified`` headers to the response if they are not already set by the view and the request method is safe."